.travis.yml

```diff
@@ -40,15 +41,36 @@ matrix:
       addons: {"apt":{"sources":["hvr-ghc"],"packages":["ghc-8.2.2","cabal-install-2.4"]}}
     - compiler: ghc-8.0.2
       addons: {"apt":{"sources":["hvr-ghc"],"packages":["ghc-8.0.2","cabal-install-2.4"]}}
+  allow_failures:
+    - compiler: ghc-8.8.1
 before_install:
+  - |
+    if [ "$TRAVIS_OS_NAME" = "linux" ]; then
+      sudo add-apt-repository -y ppa:hvr/ghcjs;
+      curl -s https://deb.nodesource.com/gpgkey/nodesource.gpg.key | sudo apt-key add -
+      sudo apt-add-repository 'https://deb.nodesource.com/node_8.x xenial main'
+      sudo apt-get update;
+      sudo apt-get install $CC cabal-install-3.0 nodejs;
```

Comment:
Shouldn't some of this be conditional on `$GHCJS`? It seems to be with the latest `haskell-ci`.

Reply:
Yes. Fixed.